Makefile

```diff
@@ -6,7 +6,11 @@ all: cpp java python
 
 .PHONY: clean all alltests clean \
 	java java_test java_clean cpp cpp_test cpp_clean \
-	python python_test	
+	python python_test
+
+docker: cpp cpp_test
+	sudo docker build -t certificate_transparency/super_duper:test .
```

Comment:
I put it here because it's building from the "source" that's also in this dir, seems to be a common pattern to have a `Dockerfile` at the root.  Technically I guess it could go elsewhere, but then you'd have to provide a path back to here anyway for the `ADD`/`COPY` commands in the `Dockerfile` to work (the etcd `Dockerfile` can go anywhere without worrying about the path because it doesn't use any local resources.)